File: Clasificados/TypeScript/models/inmueble.ts
export enum VentaRenta {
    Venta,
    Renta
};
export enum CasaTerreno {
    Casa,
    Terreno
};
export interface Inmueble {
    id: number;
    estadoId: number;
    ciudadId: number;
    descripcion: string;
    fecha: Date;
    activo: boolean;
    ventaRenta: VentaRenta;
    casaTerreno: CasaTerreno;
}

File: Clasificados/Models/Inmueble.cs
using System;

namespace Clasificados.Models
{
    public enum VentaRenta
    {
        Venta,
        Renta
    }

    public enum CasaTerreno
    {
        Casa,
        Terreno
    }

    public class Inmueble : AnuncioBase
    {
        public VentaRenta VentaRenta { get; set; }
        public CasaTerreno CasaTerreno { get; set; }
    }
}
